CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # NHS.UK prototype kit Changelog
 
+## Unreleased
+
+:new: **New features**
+
+- Update to NHS.UK frontend v9.0.0
```

Comment:
Add link to the release?

Reply:
done in 3baa3468081e9d0a0dfd9bd080a98cf2e6cfaedb